tests/integration/targets/inventory_aws_ec2/templates/inventory_with_include_or_exclude_filters.yml.j2

```
@@ -0,0 +1,21 @@
+plugin: amazon.aws.aws_ec2
+aws_access_key_id: '{{ aws_access_key }}'
+aws_secret_access_key: '{{ aws_secret_key }}'
+{% if security_token | default(false) %}
+aws_security_token: '{{ security_token }}'
+{% endif %}
+regions:
+- '{{ aws_region }}'
+filters:
+  tag:Name:
+  - '{{ resource_prefix }}_1'
+  - '{{ resource_prefix }}_3'
+include_filters:
+- tag:Name:
+  - '{{ resource_prefix }}_2'
+exclude_filters:
+- tag:Name:
+  - '{{ resource_prefix }}_3'
```

While the logic paths are currently the same, I'd recommend adding an _4 host too.  To handle someone messing with the logic in future.

```suggestion
  - '{{ resource_prefix }}_2'
  - '{{ resource_prefix }}_4'
exclude_filters:
- tag:Name:
  - '{{ resource_prefix }}_3'
  - '{{ resource_prefix }}_4'
```